source/connect.txt

```diff
@@ -94,6 +96,97 @@ Replica Set
    
    try (MongoClient mongoClient = MongoClients.create(uri))
 
+Sharded Cluster
+~~~~~~~~~~~~~~~
+
+To connect to a sharded cluster, specify the ``mongos`` instance or
+instances to the ``MongoClients.create()`` method. To learn more about
+sharded clusters, see :manual:`Sharding </sharding/>` in the Server
+manual.
+
+You can connect to a single ``mongos`` instance in the following ways:
+
+- Specify the hostname and the port in a ``ConnectionString``:
```

Comment:
S: introduce the code example. Also, I don't think ConnectionString needs to be monospaced:

```suggestion
- Specify the hostname and the port in a connection string, as shown in the following code:
```

Reply:
Applies to all (both the introductory phrase and the connection string formatting)